### ISSUE TITLE
Example of the project doesn't work for me

### ISSUE THREAD

**Issue by Dersh** [jonsamwell/flutter_gherkin#189]
I'm running on Android and get the error.
The app is just hanging on the white screen.

```
[ +202 ms] E/flutter (13121): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled
Exception: Exception: Error when trying to find feature files with patternsRegExp:
pattern=features/*.*.feature flags=`ERROR: `FileSystemException: Directory listing
failed, path = '/' (OS Error: Permission denied, errno = 13)`
[        ] E/flutter (13121): #0      GherkinRunner._getFeatureFiles
(package:gherkin/src/test_runner.dart:130:7)
[        ] E/flutter (13121): <asynchronous suspension>
[        ] E/flutter (13121): #1      GherkinRunner.execute
(package:gherkin/src/test_runner.dart:43:24)
[        ] E/flutter (13121): <asynchronous suspension>
[        ] E/flutter (13121): 
```

Flutter doctor:
[✓] Flutter (Channel stable, 2.5.3, on macOS 11.5.2 20G95 darwin-arm, locale en-RU)
    • Flutter version 2.5.3 at /Users/dersh/sdk/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (9 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/dersh/Library/Android/sdk
    • Platform android-31, build-tools 29.0.2
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.63.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.29.0

[✓] Connected device (3 available)
    • SM A715F (mobile)                    • R58NB1SJ64B   • android-arm64  •
      Android 11 (API 30)
    • Android SDK built for arm64 (mobile) • emulator-5554 • android-arm64  •
      Android 12 (API 31) (emulator)
    • Chrome (web)                         • chrome        • web-javascript • Google
      Chrome 96.0.4664.110

**Comment by jonsamwell** [comment]
This looks like a permission issue see the error `ERROR: `FileSystemException: Directory listing
failed, path = '/' (OS Error: Permission denied, errno = 13)`` what version of this library are you using?

**Comment by AFASbart** [comment]
This is indeed a permission / configuration error. I have started the example project but I am getting errors of an outdated version of Kotlin / Android SDK. I think it should be a good idea to create a version with the latest versions? @jonsamwell

**Comment by jonsamwell** [comment]
Should be fixed in the latest release `3.0.0-rc.10`
